project/Merging.scala

```diff
@@ -15,6 +15,7 @@ object Merging {
     case x if x.endsWith("kotlin-stdlib.kotlin_module") => MergeStrategy.first
     case x if x.contains("bouncycastle") => MergeStrategy.first
     case x if x.endsWith("kotlin-stdlib-common.kotlin_module") => MergeStrategy.first
+    case x if x.endsWith("okio.kotlin_module") => MergeStrategy.first
```

Comment:
this fixes an issue in `sbt assembly`, which we run as part of deployment and tests, but developers typically do not run locally. `sbt assembly` compiles the [fat jar](https://www.baeldung.com/scala/sbt-fat-jar), and hit an error trying to de-duplicate `okio-jvm META-INF/okio.kotlin_module` and `okio META-INF/okio.kotlin_module`. This fix takes the same approach as the merge strategy for `kotlin-stdlib` and `otlin-stdlib-common` just above in this file.